packages/moonstone/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ The following is a curated list of changes in the Enact moonstone module, newest
 
 ## [unreleased]
 
+### Changed
+
+- `moonstone/Picker` to change `max-width` of valueWrapper
```

Comment:
We try to phrase change log entries in terms of the effect on the developer or user rather than the technical change that was made. We also generally reserve `Changed` for significant behavior or API changes that may require developer action to adopt.

I usually try to answer the question, "What problem might I have seen in my application that would no longer be present?" In this case, maybe something like:

> ### Fixed
>
> - `moonstone/Picker` to display more of the selected value in wide instances